physics/body_centred_body_direction_dynamic_frame_test.cpp

```diff
@@ -208,9 +208,8 @@ TEST_F(BodyCentredBodyDirectionDynamicFrameTest, CoriolisAcceleration) {
                             20 * Metre / Second,
                             30 * Metre / Second})};
   DegreesOfFreedom<ICRS> const big_dof =
-      {Displacement<ICRS>({0 * Metre, 0 * Metre, 0 * Metre}) +
-           ICRS::origin,
-       Velocity<ICRS>()};
+      {Displacement<ICRS>({0 * Metre, 0 * Metre, 0 * Metre}) + ICRS::origin,
+       ICRS::unmoving};
```

Comment:
Let's simplify that to `{ICRS::origin, ICRS::unmoving}`, I don't think adding 0 adds much to clarity here.

Reply:
Done everywhere in this file.